packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -75,6 +75,22 @@ export interface ICheckboxProps extends React.ButtonHTMLAttributes<HTMLElement |
   theme?: ITheme;
 
   /**
+   * Accessible label for the checkbox. Defaults to 'label' property value.
```

Comment:
> Are these props necessary?

Yes, ButtonHTMLAttributes doesn't have definition of `aria-*` attributes. Another way would be to have an interface inherited from ButtonHTMLAttributes and that contains additional `aria-*`. I like the `aria-attributes` to be at control level to make them more visible.

> I don't see code that assigns the label's value to this property by default?

`htmlFor` on the label takes care of linking the label with control.